### PR TITLE
Download weights from huggingface to help us save bandwith 

### PR DESCRIPTION
**Pull request by Jainam2130**
The torrent seed is extremely slow this should definitely help out

### CODE REVIEW

**Comment by Jainam213** [comment]
The torrent seed is extremely slow this should definitely help out

**Comment by andrewssobral** [comment]
image

**Comment by xuuyz** [comment]
This will save so much bandwidth. Thanks!

**Review by ChristopherKing42 — APPROVED** [review]

**Comment by ChristopherKing42** [comment]
LGTM

**Comment by LJ3D** [comment]
:rocket: LGTM :rocket: :rocket:

**Comment by xloem** [comment]
it’s a dataset rather than a model?

**Comment by Jainam213** [comment]
Its uploaded as a dataset but check our the files the weights are there

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 This repository is intended as a minimal, hackable and readable example to load [LLaMA](https://ai.facebook.com/blog/large-language-model-llama-meta-ai/) ([arXiv](https://arxiv.org/abs/2302.13971v1)) models and run inference.
 In order to download the checkpoints and tokenizer, fill this [google form](https://forms.gle/jk851eBVbX1m5TAv5)
+Alternatively you can dowanload them from huggingface to help us save bandwidth
```
Comment (andrewssobral) [inline README.md:5]: typo

**Comment by Blues-star** [comment]
The repository shows that it has been lost. Is there a problem?

**Comment by Daviljoe193** [comment]
Unlike torrents, Huggingface repos arent immutable, and Meta has been pressuring HF to remove repos that contain LLaMA models, so just try focusing effort on seeding the main torrent.

**Comment by pauldog** [comment]
Just to be clear. There is no legal reason why Huggingface removed these weights. (As weights are not copyrightable). They are just doing this to be on the good side of Meta.

**Comment by Nil-Andreu** [comment]
How could be loaded a pytorch model if we had separated pickle files for the weights?

**Comment by niansa** [comment]
I guess uploading to other sources may be an option too :smile:
Unfortunately most people don't have servers with a connection beefy enough to support this.

**Comment by errolyan** [comment]
LGTM

**Review by nuhmanpk — APPROVED** [review]

**Comment by nuhmanpk** [comment]
image

**Comment by pauldog** [comment]
😮 😸
2602835e667803df42dbcc551d8762fd

**Comment by msaroufim** [comment]
LLAMA 2 weights are now on HF

**Pull request closed without merging**